cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -1351,16 +1349,44 @@ func (p *BicepProvider) createOutputParameters(
 	return outputParams
 }
 
+type parameterFileContent struct {
+	fileType bicepParameterType
+	content  []byte
+	filePath string
+}
+
+// parameterFile finds the type of parameter used by the module.
+func (p *BicepProvider) parameterFile(ctx context.Context) (*parameterFileContent, error) {
+	possibleFiles := p.parametersTemplateFilePath()
+	for pType, file := range possibleFiles {
```

Comment:
In go, the order of enumeration over a map is random.  In the case where you have both a `.bicepparams` and a `parameters.json` file, I think that `parameterFile` will become nondeterministic.  I suspect we should just pick a fixed order and enforce it here?